code/office-director/office-director.js

```diff
@@ -69,3 +79,26 @@ $(document).on('knack-view-render.any', function(event, view, data) {
  $("a.kn-view-asset").html("Attachment"); 
  
 });
+// set random password when adding an account.        //
+// the user will not use this pw. they login with ADFS    //
+//////////////////////////////////////////////////////////////
+
```

Comment:
Instead of using slashes for a comment section, use `/*` comment `*/` this way you don't have to `//` multiple lines

This also makes it easier to distinguish temporary comments if you use them.